pkgs/top-level/all-packages.nix

```diff
@@ -28361,6 +28361,7 @@ with pkgs;
     ubootSopine
     ubootUtilite
     ubootWandboard
+    ubootRock4CPlus
```

Comment:
Sorry for continuing to nitpick, but this should be moved alphabetically too.

Reply:
My bad, forgot that one :facepalm: